src/stream/src/executor/wrapper/trace.rs

```diff
@@ -48,9 +48,9 @@ pub async fn trace(
             if chunk.cardinality() > 0 {
                 metrics
                     .executor_row_count
-                    .with_label_values(&[&actor_id_string, &info.identity])
+                    .with_label_values(&[&actor_id_string, &span_name])
```

Comment:
span_name seems to contain strange numbers(`100000000C`), is this expected behavior?
Or we can remove this info from grafana dashboard.
<img width="464" alt="image" src="https://github.com/risingwavelabs/risingwave/assets/58715567/2e4bd420-1df2-451b-8ca9-235294b75cd8">

Reply:
This is the executor id in hex format. Not sure if it is useful for debugging.

Reply:
I notice that we have show executor id outside it , and maybe it's outdated. I have try to remove `executor_id` from identity, but too many files changed🥲. Let's keep this for now.

Reply:
The id we shown "outside" is actor id and operator id, while the executor id is a combination of them. But it's still right that a pair of actor id and operator id is enough for locating an executor.

Reply:
BTW, the identity should actually be the `Display` format of the corresponding frontend plan node. See https://github.com/risingwavelabs/risingwave/issues/5940.